src/PantherActions.php

```diff
@@ -267,7 +280,7 @@ protected static function findFormField(string $fieldText, string $legend = null
     {
         $xpath = <<<'XPATH'
             [
-                contains(concat(' ', normalize-space(text()), ' '), %1$s) or
+                contains(concat(' ', normalize-space(string(.)), ' '), %1$s) or
```

Comment:
`text()` only uses the direct `textContent`, which does not work if you have a `<span>` inside your `<label>`, for example.